include/vrv/view.h

```diff
@@ -487,7 +488,7 @@ class View {
         wchar_t start = 0, wchar_t end = 0);
     void DrawSmuflString(DeviceContext *dc, int x, int y, std::wstring s, bool center, int staffSize = 100,
         bool dimin = false, bool setBBGlyph = false);
-    void DrawLyricString(DeviceContext *dc, int x, int y, std::wstring s, int staffSize = 100);
+    void DrawLyricString(DeviceContext *dc, TextDrawingParams &params, std::wstring s, int staffSize = 100);
```

Comment:
Were all references to `View::DrawLyricString` changed to use `TextDrawingParams`?